libs/docs/core/shellbar/examples/shellbar-select-example.component.html

```diff
@@ -0,0 +1,31 @@
+<div>
```

Comment:
Why the div?

Reply:
oh yeah, it might left when removed size segmented buttons.